README.md

```diff
@@ -28,7 +28,7 @@ This workshop is focused on deploying a fully functional web application server
 
 The workshop is divided into the following sections:
 
-1. Bulding your first GraphQL server using AeroGear Graphback
+1. Bulding Your first GraphQL Server using AeroGear Graphback and AeroGear Voyager-Server
```

Comment:
I think its building. I would also stay with "your" not "Your" as this is a tutorial only not a formal letter.